### DISCUSSION TITLE
LSTM-pool run time

### DISCUSSION THREAD

**Discussion by ruvyanet** [asreview/asreview#1242] [Q&A]
I’m currently running simulations on a subset of articles (n = 1600) from the 28,000 pulled for my systematic review. The simulation with classifier LTSM-pool has been running for three days, is it normal for this simulation to be running for so long? I’m worried something is wrong, and I would greatly appreciate your insight and expertise on this topic.  And if it's taking this long for a subset of the sample during the simulation, should I expect it to take exponentially longer for the larger dataset?

I appreciate your consideration.  

Kindly,
Yanet

**Comment by jteijema** [comment]
Hi @ruvyanet!

The LSTM feature extraction and LSTM classifiers are very complex models, and require "heavy" equipment to run, but haven't shown to improve performance over other classifiers that are available. For this reason, I personally refrain from using this classifier, for my own studies.

Can I ask why you picked this setting? Maybe we can find a different classifier that is more appropriate for your application.

^Jelle

**Reply by ruvyanet** [reply]
Hi Jelle, 

Thank you for explaining! 

The plan was to run simulations with the sample dataset, review the performance metrics, choose the best performing machine learning classifier and data extraction combination, and then run the entire dataset in ASReview with the chosen machine learning model components. 

Best,
Yanet

**Reply by Rensvandeschoot** [reply]
Have you tried the `multiple-model`-template of [Makita](https://github.com/asreview/asreview-makita#multiple-models-template)?

**Reply by ruvyanet** [reply]
No, but I can definitely look into it. Thanks!

**Comment by J535D165** [comment]
See https://asreview.readthedocs.io/en/latest/simulation_cli.html#cmdoption-n_instances. We usually bump this to 50 or 100 for heavy models.

**Reply by ruvyanet** [reply]
Thank you!!

**Reply by J535D165** [reply]
It also helps to use version 1 of ASReview (if you are still at 0.x). Version 1 has an improved data model, and https://asreview.readthedocs.io/en/latest/simulation_cli.html#cmdoption-stop_if is set to min by default (huge performance gain).
